config/index.js

```diff
@@ -3,6 +3,10 @@
 const packageJson = require('../package.json');
 
 module.exports = {
+  env: process.env.NODE_ENV,
```

Comment:
Why have you added these, are they needed?

Reply:
Discussed and resolved